tests_e2e/scripts/collect_logs.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
executed on the test VM to collect agent and system logs into a compeessed tarball